outbound/state/work_description.py

```diff
@@ -0,0 +1,132 @@
+import utilities.integration_adaptors_logger as log
+from edifact.outgoing.models.interchange import InterchangeHeader, InterchangeTrailer
+from edifact.outgoing.models.message import MessageHeader, MessageTrailer, ReferenceTransactionNumber, \
+    ReferenceTransactionType
+from utilities.message_utilities import get_uuid
+
+from persistence import persistence_adaptor as pa
+
+DATA = 'DATA'
+OPERATION_ID = 'OPERATION_ID'
+TRANSACTION_ID = 'TRANSACTION_ID'
+TRANSACTION_TIMESTAMP = 'TRANSACTION_TIMESTAMP'
+TRANSACTION_TYPE = 'TRANSACTION_TYPE'
+SIS_SEQUENCE = 'SIS_SEQUENCE'
+SMS_SEQUENCES = 'SMS_SEQUENCES'
+SENDER = 'SENDER'
+RECIPIENT = 'RECIPIENT'
+
+logger = log.IntegrationAdaptorsLogger(__name__)
+
+
+class WorkDescription(object):
```

Comment:
Perhaps rename WorkDescription to OutboundState or OutboundTransactionState? The concept of a "work description" is specific to MHS I think.

Reply:
OutboundTransactionState